File: post/model/reaction.go
package model

import (
	"go.mongodb.org/mongo-driver/bson/primitive"
	"time"
)

type Reaction struct {
	ReactionID     primitive.ObjectID `bson:"_id" json:"reactionId"`
	PostID         primitive.ObjectID `json:"postId"`
	UserEmail      string             `json:"userEmail"`
	CreationTime   time.Time          `json:"creationTime"`
	TypeOfReaction ReactionType       `json:"reactionType"`
}

File: DislinktFrontApp/src/app/model/reaction.ts
export class Reaction{
    reactionId:number;
	postId:number;
	userId:number;
	creationTime:any;
	reactionType:string;

    constructor(reactionId:number,
        postId:number,
        userId:number,
        creationTime:any,
        reactionType:string){
            this.reactionId=reactionId;
            this.postId=postId;
            this.userId=userId;
            this.creationTime=creationTime;
            this.reactionType=reactionType;
        }
}